src/mococrw/openssl_wrap.h

```diff
@@ -1412,5 +1427,17 @@ std::vector<uint8_t> _i2d_ECSDA_SIG(const ECDSA_SIG*);
  */
 SSL_BIGNUM_Ptr _BN_bin2bn(const uint8_t* data, size_t dataLen);
 
+/**
+ * Write an OpenSSL bignum in big-endian plain bytes representation of an
+ * unsigned integer.
+ *
+ * @param bignum Number to write to
+ * @param to Array of chars to write the number to
+ * @param tolen Length of to
+ * @return Number of bytes written into to
+ * @throw OpenSSLException if the supplied buffer is too small
+ */
+int _BN_bn2binpad(const BIGNUM *bignum, unsigned char *to, int tolen);
```

Comment:
modified the `_BN_bn2binpad` function. However i skipped the resize of the vector because especially in the current usecase it is important that out has exactly the length of the key size (because that is the IEEE signature format). In addition, it seems that vectors can not be concatenated using the + operator...